src/vision.cs

```diff
@@ -150,6 +159,31 @@ interface VNCoreMLRequest {
 		[Export ("initWithCompletionHandler:")]
 		[DesignatedInitializer]
 		IntPtr Constructor ([NullAllowed] VNRequestCompletionHandler completionHandler);
+
+		// We must inline the following static properties so the correct class_ptr is used.
```

Comment:
I wonder if this is something we could add support for in the generator.

Reply:
Yeah, I was thinking on adding a flag to [Static] or creating a new attribute for it since sometimes you want the return type to differ and sometimes you dont.